tests/components/modern_forms/test_config_flow.py

```diff
@@ -223,16 +217,6 @@ async def test_zeroconf_with_mac_device_exists_abort(
     """Test we abort zeroconf flow if a Modern Forms device already configured."""
     await init_integration(hass, aioclient_mock, skip_setup=True)
 
-    await hass.config_entries.flow.async_init(
-        DOMAIN,
-        context={"source": SOURCE_USER},
```

Comment:
I have remove this, as it is no longer needed since the `unique_id` is correctly set in `init_integration`